ldk/javascript/examples/consolidate-loop/README.md

```diff
@@ -0,0 +1,17 @@
+# Building and running Loop in Olive Helps
```

Comment:
Should we update this readme to include wording about what the consolidate example loop is, how to run it, and what kind of examples it provides?